src/alias/azext_alias/tests/test_alias_commands.py

```diff
@@ -30,6 +28,7 @@ def setUp(self):
         alias.GLOBAL_ALIAS_PATH = os.path.join(self.mock_config_dir, ALIAS_FILE_NAME)
         alias.GLOBAL_ALIAS_HASH_PATH = os.path.join(self.mock_config_dir, ALIAS_HASH_FILE_NAME)
         alias.GLOBAL_COLLIDED_ALIAS_PATH = os.path.join(self.mock_config_dir, COLLIDED_ALIAS_FILE_NAME)
+        util.GLOBAL_ALIAS_TAB_COMP_TABLE_PATH = os.path.join(self.mock_config_dir, ALIAS_TAB_COMP_TABLE_FILE_NAME)
         custom.GLOBAL_ALIAS_PATH = os.path.join(self.mock_config_dir, ALIAS_FILE_NAME)
```

Comment:
You should use https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch for this kind of thing rather than just money-patching the value.
Otherwise it doesn't reset properly for each test and there'll be other issues later on.